index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jaini+Purva&display=swap" rel="stylesheet">
+    <title>Rats</title>
+</head>
+
+<body>
+    <header>
+
+    </header>
+    
+    <h1>
+        My Favortie Animals are Rats
+    </h1>
+    <h2>Domesticated Rats</h2>
+    <div class="row">
+        <div class="column">
+    <h3>Rat History</h3>
+    <p>The domesticated rat has uncertain origins. The first evicendce of rats being kept as pets is as early as the Edo
+        period (1603-1868) in Japan. The earliest recorded breeding of rats dates back to 1654. </p>
+        <br>
+        <p>Rats being bred for food and sport began in the early 1800's in England, France, and North America. Rats were used in baiting events where men would bet on how many rats a dog could kill per round. </p>
+        <br>
+        <p>Brown rats were some of the earliest to be used in laboratory experiments. By the early 1900's rat strains bred specifically for lab experiments began. It is believed that pet rats were dereived from these lab rats.</p>
+    <img src="css/pngimg.com - rat_mouse_PNG23558.png">
```

Comment:
where is your alt text? Don't store images in a CSS folder. Your image isn't showing up because of your file name.